src/lib/learn-client/api/collection/formatting.ts

```diff
@@ -71,6 +72,7 @@ export function formatToTutorialLite(
 	return {
 		id,
 		name,
+		shortName: short_name || name,
```

Comment:
Aligning with the camelCase formatting convention. Once short name is required and all the data is populated by edu, we won't default to `name`